crates/graphql/src/schema/objects/nft.rs

```diff
@@ -16,6 +18,10 @@ use scalars::{PublicKey, U64};
 use serde_json::Value;
 
 use super::prelude::*;
+use crate::schema::{
+    dataloaders::collection::{CollectionFloorPrice, CollectionNftCount},
+    scalars::I64,
+};
```

Comment:
should be able to reference without the use create::schema

Just `use scalars::I64` and `use dataloaders::collection::{...}`